source/platforms/coiled.md

```diff
@@ -1,65 +1,142 @@
 
 By using the [`coiled`](https://anaconda.org/conda-forge/coiled) Python library, you can setup and manage Dask clusters with GPUs and RAPIDs on cloud computing environments such as GCP or AWS.
 
-Coiled clusters can also run Jupyter which is useful if you don't have a local GPU.
+## Setup
 
-## Quickstart
+Head over to [Coiled](https://docs.coiled.io/user_guide/setup/index) and register for an account.
 
-### Login
+Once your account is set up, install the coiled Python library/CLI tool.
 
-To get started you need to install Coiled and login.
+```bash
+pip install coiled
+```
 
-```console
-$ conda install -c conda-forge coiled
-$ coiled login
+Then you can authenticate with your Coiled account.
+
+```bash
+coiled login
 ```
 
-For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/getting_started.html).
+For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/setup/index).
 
-### Software Environment
+## Notebook Quickstart
 
-Next you'll need to register a RAPIDS software environment with Coiled.
+The simplest way to get up and running with RAPIDS on Coiled is to launch a Jupyter notebook server using the RAPIDS notebook container.
 
-You can either build this from the official RAPIDS Container images.
+```bash
+coiled notebook start --gpu --container {{ rapids_notebooks_container }}
+```
 
-```python
-import coiled
+![Screenshot of Jupyterlab running on Coiled executing some cudf GPU code](../_static/images/platforms/coiled/coiled-jupyter.png)
 
-coiled.create_software_environment(
-    name="rapids-{{ rapids_version.replace(".", "-") }}",
-    container="{{ rapids_container }}",
-)
+## Software Environments
+
+By default when running remote operations Coiled will [attempt to create a copy of your local software environment](https://docs.coiled.io/user_guide/software/sync.html) which can be loaded onto the remote VMs. While this is an excellent feature it's likely that you do not have all of the GPU software libraries you wish to use installed locally. In this case we need to tell Coiled which software environment to use.
+
+### Container images
+
+All Coiled commands can be passed a container image to use. This container will be pulled onto the remote VM at launch time.
+
+```bash
+coiled notebook start --gpu --container {{ rapids_notebooks_container }}
 ```
 
-Or you can create it with a list of conda packages in case you want to customize the environment further.
+This is often the most convenient way to try out existing software environments, but is often not the most performant due to the way container images are unpacked.
```

Comment:
This is a fair point, but if this is the way we recommend people using it, maybe we don't want to be negative about it.

Reply:
My intent was to document that RAPIDS should either use the container, or a prebuilt software environment. It should not use package sync.

I keps the container in most of the examples because it feels simpler and is very copy/paste friendly. But the prebuilt software environment is more performant, and it probably my recommendation as a best practice.

What do you think we should do to make this more clear?

Reply:
I agree that package sync is more performant, but that implies that you have
1. a properly set environment, which is not hard
2. a GPU, if you don't package sync will not work

My guess is that someone trying to use coiled, probably doesn't have a local GPU in their laptop. I might be wrong, though. 

What we could do, is mentioned that to use package sync you need a local GPU and a properly set environment.

Reply:
Oh I see, I remember this now. Thanks for the clarification.